scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -68,7 +68,8 @@
     (-> k8s-actual-state-dict
         (update-in [:synthesized-state :state] #(or % :missing))
         (dissoc :pod)
-        (assoc :pod-status (some-> pod .getStatus)))
+        (assoc :pod-status (some-> pod .getStatus))
+        (assoc :node-name (api/pod->node-name pod)))
```

Comment:
### Why?

Adding the node name here makes it possible to include these logs when searching the logs for a particular node name, which is a common practice.

Reply:
Nitpick: We can merge the two `assoc`s

Reply:
Good call, thanks

Reply:
Done!